src/api/ApiClient.js

```diff
@@ -0,0 +1,36 @@
+import {apiBaseUrl} from "../tools"
+
+class ApiClient {
+    constructor() {
+        this.baseUrl = apiBaseUrl
+    }
+
+    async get(url) {
+        return fetch(`${this.baseUrl}${url}`)
```

Comment:
```suggestion
        return fetch(`${this.baseUrl}/${url}`)
```
Plutot non ?

(pareil en dessous)

Reply:
ApiClient.get(/collaborators)

ApiClient.get(collaborators)

je préfère la 1 perso

Reply:
C'est bizarre non ?

@FeryKevin avis ?